src/main/java/org/isf/menu/gui/UserEdit.java

```diff
@@ -247,6 +247,16 @@ private JButton getOkButton() {
 						MessageDialog.error(null, "angal.userbrowser.passwordsdonotmatchpleasecorrect.msg");
 						return;
 					}
+					String passwordStr = new String(password);
+					if (!manager.isPasswordStrong(passwordStr)) {
+						MessageDialog.error(null, "angal.userbrowser.passwordsmustcontainatleastonealphabeticnumericandspecialcharacter.msg");
+						return;
+					}
+					String hashed = BCrypt.hashpw(passwordStr, BCrypt.gensalt());
```

Comment:
it seems this variable is duplicated at line 266

Reply:
This is because this is a bad recovery from not being able to merge because of other commits.   I'm going to close this and redo it on the current develop branch